.docker/prod.Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION=unstable
+ARG VERSION=oldstable
```

Comment:
I don't think this is correct here because we wont have a `greenbone/gsad-build:oldstable. Should be edge i guess.

Reply:
I've changed it  now so the gvm-libs base image and the build image have separate versions.